crates/bws/src/main.rs

```diff
@@ -516,12 +548,38 @@ async fn process_commands() -> Result<()> {
         | Commands::Delete {
             cmd: DeleteCommand::Secret { secret_ids },
         } => {
-            client
+            let result = client
                 .secrets()
                 .delete(SecretsDeleteRequest { ids: secret_ids })
                 .await?;
 
-            println!("Secret deleted correctly");
+            let mut secrets_success = 0;
+            let mut secrets_failed: Vec<(Uuid, String)> = Vec::new();
+
+            for secret in result.data.iter() {
+                match &secret.error {
+                    Some(error_message) => {
+                        secrets_failed.push((secret.id, error_message.to_owned()))
+                    }
+                    None => secrets_success += 1,
```

Comment:
Do we need to count this? We could just get `input - failed`. Which probably lets you rewrite this section into a filter & map.

Reply:
Good point. Switched to using `filter_map` in [b8074ff](https://github.com/bitwarden/sdk/pull/77/commits/b8074ff181501627822ef320e449c7cb419b29e4)